docs/fidesops/docs/guides/configuration_reference.md

```diff
@@ -94,8 +94,9 @@ Please note: The configuration is case-sensitive, so the variables must be speci
 
  ENV Variable | Default | Description |
 |---|---|---|
-| `LOG_PII` | False | If this is set to "True", pii values will display unmasked in log output. This variable should always be set to "False" in production systems.|
-| `DEV_MODE` | False | If "True", the fidesops server will run with hot-reloading of files. This variable should always be set to "False" in production systems.|
+| `FIDESOPS_LOG_PII` | False | If this is set to "True", pii values will display unmasked in log output. This variable should always be set to "False" in production systems.
+| `FIDESOPS_HOT_RELOAD` | False | If "True", the fidesops server will run with hot-reloading of files. This variable should always be set to "False" in production systems.|
```

Comment:
I'm pretty sure `hot reloading` is a well understood term, but it might be good to add a sentence explaining that just in case?

Reply:
good suggestion